src/Cli/dotnet/commands/dotnet-test/SolutionAndProjectUtility.cs

```diff
@@ -110,7 +110,7 @@ public static IEnumerable<TestModule> GetProjectProperties(string projectFilePat
 
         var targetFramework = project.GetPropertyValue(ProjectProperties.TargetFramework);
         var targetFrameworks = project.GetPropertyValue(ProjectProperties.TargetFrameworks);
-        Logger.LogTrace(() => $"Loaded project '{Path.GetFileName(projectFilePath)}' with TargetFramework '{targetFramework}', TargetFrameworks '{targetFrameworks}', IsTestProject '{project.GetPropertyValue(ProjectProperties.IsTestProject)}', and '{ProjectProperties.IsTestingPlatformApplication}' is '{project.GetPropertyValue(ProjectProperties.IsTestingPlatformApplication)}'.");
+        Logger.LogTrace($"Loaded project '{Path.GetFileName(projectFilePath)}' with TargetFramework '{targetFramework}', TargetFrameworks '{targetFrameworks}', IsTestProject '{project.GetPropertyValue(ProjectProperties.IsTestProject)}', and '{ProjectProperties.IsTestingPlatformApplication}' is '{project.GetPropertyValue(ProjectProperties.IsTestingPlatformApplication)}'.");
```

Comment:
Before the change, if logging wasn't enabled, we will have small allocations here.
With the change, this will allocate nothing if logging isn't enabled.

We could have prevented the allocation by wrapping the LogTrace in a TraceEnabled condition, but I find it easier to have that centralized in the logger itself.